bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -474,6 +474,17 @@ pub enum TimelineItemContentKind {
         info: ImageInfo,
         url: String,
     },
+    Poll {
+        question: String,
+        kind: PollKind,
+        max_selections: u64,
+        answers: Vec<PollAnswer>,
+        votes: HashMap<String, Vec<String>>,
+        end_time: Option<u64>,
+    },
+    PollEnd {
+        start_event_id: String,
+    },
```

Comment:
What's the idea here? When the poll ends, what should be rendered in the timeline?

Reply:
> What's the idea here? When the poll ends, what should be rendered in the timeline?

The eventual goal is for something that looks similar to a state event (i.e. just a single line of text not in a bubble that says the poll is ended) that, when tapped, takes you up in the timeline to the corresponding start event. [I have a feeling the "when tapped" part is either not feasible or is non-trivial at the moment though, so the first implementation probably won't have that]

In EX we're explicitly _not_ going to duplicate the poll state when receiving an end event like legacy EA/EI do

Reply:
> so you can se which poll ended before clicking that button

In the initial version, no. I don't think we can reliably get any other data (such as the question) without using something like the relations API, and we're not doing that for now

Reply:
Hm, I see what you mean, that would mandate a more complicated implementation than what we need for initial support. Alright.